automation/README.md

```diff
@@ -0,0 +1,70 @@
+# Automation with oVirt CI
+
+In order to keep playbooks in this repository operational,
+it is being integrated with [oVirt CI System][ovirt-ci-system-doc].
+Everything located under `automation` directory is related to integration
+to [oVirt CI System][ovirt-ci-system-doc].
+
+There is a list of events which can trigger a job on [oVirt CI System][ovirt-ci-system-doc],
+The KubeVirt Ansible repository uses the `check-patch` event at the moment,
+to verify incoming patches.
+
+Once you submit a PR for this repository, the [oVirt CI System][ovirt-ci-system-doc]
+triggers [Jenkins job](http://jenkins.ovirt.org/blue/organizations/jenkins/kubevirt_kubevirt-ansible_standard-check-pr/activity)
+which executes [`automation/check-patch.sh`](./check-patch.sh) script.
+
+This script will execute Ansible playbook [`./control.yml`](../control.yml)
+which wraps entire testing flow.
+
+
+## `control.yml` Ansible playbook
+
+This Ansible playbook is wrapper for entire testing flow which is composed from
+following steps (partial playbooks)
+* [provision testing environment](#testing-environment)
+* [deploy OpenShift](#deploy-openshift)
+* [install KubeVirt](#install-kubevirt)
+* [TODO](https://github.com/kubevirt/kubevirt-ansible/issues/47) run KubeVirt test suite
+
+This playbook is executed by Ansible 2.4.2.
+
```

Comment:
s/is executed by/requires

Reply:
I want to express that the testing envirionment is really using this specific version.
But required is Ansible >= 2.4 for instance.

I can rephrase that somehow ...

Reply:
Rephrased